src/widgets/Window.cpp

```diff
@@ -642,31 +642,38 @@ void Window::addShortcuts()
 
              if (arg == "off")
              {
-                 this->notebook_->setShowTabs(false);
-                 getSettings()->tabVisibility.setValue(
-                     NotebookTabVisibility::AllTabs);
+                 this->notebook_->hideAllTabsAction->trigger();
              }
              else if (arg == "on")
              {
-                 this->notebook_->setShowTabs(true);
-                 getSettings()->tabVisibility.setValue(
-                     NotebookTabVisibility::AllTabs);
+                 this->notebook_->showAllTabsAction->trigger();
              }
              else if (arg == "toggle")
              {
-                 this->notebook_->setShowTabs(!this->notebook_->getShowTabs());
-                 getSettings()->tabVisibility.setValue(
-                     NotebookTabVisibility::AllTabs);
+                 this->notebook_->toggleTabVisibility();
              }
              else if (arg == "liveOnly")
              {
-                 this->notebook_->setShowTabs(true);
-                 getSettings()->tabVisibility.setValue(
-                     NotebookTabVisibility::LiveOnly);
+                 this->notebook_->onlyShowLiveTabsAction->trigger();
              }
              else if (arg == "toggleLiveOnly")
              {
-                 this->notebook_->toggleOfflineTabs();
+                 // NOOP: Removed 2024-08-04
+                 auto *selectedPage = this->notebook_->getSelectedPage();
+                 if (selectedPage != nullptr)
+                 {
+                     auto *selectedSplit = selectedPage->getSelectedSplit();
+                     if (selectedSplit != nullptr)
+                     {
+                         auto selectedChannel = selectedSplit->getChannel();
+                         if (selectedChannel != nullptr)
+                         {
+                             selectedChannel->addSystemMessage(
+                                 "setTabVisibility's toggleLiveOnly has been "
+                                 "removed");
+                         }
+                     }
+                 }
```

Comment:
Update the code below to no longer include toggleLiveOnly.

Reply:
:+1: 
Fixed in [`8b5c532` (#5530)](https://github.com/Chatterino/chatterino2/pull/5530/commits/8b5c53234f599cc8bb5ca0a579a6dda33b5aea1e)